app/src/main/java/ai/elimu/analytics/entity/NumberLearningEvent.kt

```diff
@@ -0,0 +1,15 @@
+package ai.elimu.analytics.entity
+
+import ai.elimu.model.v2.enums.analytics.LearningEventType
+import androidx.room.Entity
+
+@Entity(tableName = "NumberLearningEvent")
```

Comment:
```suggestion
```

@tuancoltech We are not setting the table name in any of the other entities. So for consistency, it would be better if you don't include it here.